spring-kafka/src/main/java/org/springframework/kafka/retrytopic/RetryTopicConfigurationBuilder.java

```diff
@@ -93,6 +95,11 @@ public RetryTopicConfigurationBuilder dltHandlerMethod(String beanName, String m
 		return this;
 	}
 
+	public RetryTopicConfigurationBuilder concurrency(Integer concurrency) {
```

Comment:
I know most of the methods in this class don't have proper javadocs, but perhaps we should at least include in the new ones?

Reply:
BTW, do you think it'd be worth it adding javadocs for the other methods as well? I can open a PR with that if you'd like.